packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 0.10.4+2
 
 * Updates compileSdkVersion to 33.
+* Fixes false positive for CamcorderProfile deprecation warning
```

Comment:
This will need to be updated for the new scope of the PR.

Reply:
Done